sale_product_deposit/data/account_tax_group_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="1">
+        <record id="deposit_tax_group" model="account.tax.group">
+            <field name="name">Deposit</field>
```

Comment:
since the value of the name field changed from french to english, it will require a translation, but since this record is set as `noupdate`, it will not be a problem for instances where it is already installed.